experiments/AMIP/Manifest.toml

```diff
@@ -1,6 +1,6 @@
 # This file is machine-generated - editing it directly is not advised
 
-julia_version = "1.10.0"
+julia_version = "1.9.3"
```

Comment:
we don't want to downgrade to julia v1.9.3 - could you upgrade to 1.10 then resolve the manifest?

Reply:
Ahh bad rebase! Good catch!!